browser_history/cli.py

```diff
@@ -27,6 +27,18 @@ def make_parser():
                          help=f'''
                                 browser to retrieve history from. Should be one of all, {AVAILABLE_BROWSERS}.
                                 Default is all (gets history from all browsers).''')
+
+    parser_.add_argument('-f', '--format',
+                         default="csv",
+                         help=f'''
+                                Format to be used in output. Should be one of {generic.Outputs.formats}.
```

Comment:
This looks a little weird in the help page (`--help`).
```
  -f FORMAT, --format FORMAT
                        Format to be used in output. Should be one of ('csv',). Default is CSV
```

You could format it like in `--browser`.

Reply:
lemme patch it up real quick

Reply:
I have made the change. The message seems a little weird because csv is the only format available but it should make sense once we add other formats.

Reply:
Yes, that's ok. We won't be making a release until atleast JSON has been added.